owtf/webapp/src/containers/Report/Table.js

```diff
@@ -212,6 +224,25 @@ export class DataTable extends React.Component {
           <Pane border padding={10}>
             <Heading>MORE DETAILS</Heading>
             {/* {output} */}
+            { /**Display only when resourcelist is more than 0,else output.Resourcelist = undefined will appear in UI  */
+              (resourceList.length > 0) &&
```

Comment:
In case of an empty list, a message (like "Output not available") would be more useful.

Reply:
Thank You, Will make the changes